### ISSUE TITLE
after Longpress(Draggable item) how can i find out that item is moved or not?

### ISSUE THREAD

**Issue by BlackAngel1991** [h6ah4i/android-advancedrecyclerview#191]
i want know how can i find out that after entering in draggable  state with OnLongPress, item moved or not,
i know i cant check item position in `setOnItemDragEventListener` but i want to check item top position is changed or not.

**Comment by h6ah4i** [comment]
Hi. Why do you need to know dragging items's top position? Is it not enough to check whether the `onItemDragPositionChanged()` callback is called or not?

**Comment by BlackAngel1991** [comment]
hi @h6ah4i. `onItemDragPositionChanged` callback  only called when 2 items swap operation happen.
for example if user drag an item, when he drag it over another item `onItemDragPositionChanged` will be called but if he just move it a little (not over another item), `onItemDragPositionChanged` wont be called.
as i mentioned in my last question ( #187 ) i want to do this :
after item is released,if its not moved,  item layout will be changed and user can edit item.but if its not moved nothing will happen.
i don't have any problem in other part of this operation, only problem is i cant find out item is moved or not.

**Comment by h6ah4i** [comment]
OK. I'll try to add a new callback to `OnItemDragEventListener` which can identify the distance of currently dragging item is moved.

**Comment by h6ah4i** [comment]
@BlackAngel1991 

I have just pushed the [`feature/add_on_item_drag_move_distance_callback`](https://github.com/h6ah4i/android-advancedrecyclerview/tree/feature/add_on_item_drag_move_distance_callback) branch. I added a new callback `onItemDragMoveDistanceUpdated` to `OnItemDragEventListener`.

Its ready to use through JitPack now! Please modify your `build.gradle` file to use the new feature. Thanks :wink: 

#### OnItemDragEventListener
```java
    public interface OnItemDragEventListener {
        /**
         * Callback method to be invoked when the distance of currently dragging item is updated.
         *
         * @param offsetX The horizontal distance of currently dragging item from the initial position in pixels.
         * @param offsetY The vertical distance of currently dragging item from the initial position in pixels.
         */
        void onItemDragMoveDistanceUpdated(int offsetX, int offsetY);
    }
```

#### build.gradle
```gradle
repositories {
    maven { url "https://jitpack.io" }
}

dependencies {
    compile 'com.github.h6ah4i:android-advancedrecyclerview:c8cb75c79ccaa97f78ba75749bb76432e31c8302'
}
```

**Comment by BlackAngel1991** [comment]
ty @h6ah4i. that's exactly what i needed, again you are awesome.

**Comment by h6ah4i** [comment]
Yes. This feature will be merged into the main branch.

**Comment by h6ah4i** [comment]
@BlackAngel1991 Hi. I have just released v0.8.7 and it includes OnItemDragEventListener support. Thanks.
